docker-compose.yml

```diff
@@ -1,12 +1,6 @@
 version: '2'
 
 services:
-  mongo:
-    image: mongo:3.5
-    container_name: sedaily-mongo
```

Comment:
Do you think we could keep the container name?

Reply:
Yep, just added it back.

Reply:
Awesome !